### PR TITLE
add InternalSampler.make_internal; rerun sampling nbs

### PR DESCRIPTION
**Pull request by yannikschaelte**

### CODE REVIEW

File: pypesto/sampling/adaptive_metropolis.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ def default_options(cls):
             'cov0': None,
             # target acceptance rate
             'target_acceptance_rate': 0.234,
+            # show progress
+            'show_progress': True,
```
Comment (elbaraim) [inline pypesto/sampling/adaptive_metropolis.py:42]: What about rename to `progress_bar`. It is more specific.

Comment (yannikschaelte) [reply pypesto/sampling/adaptive_metropolis.py:42]: Both fine for me. I thought maybe at some point one there may be other "progress indicators" aside from the progressbar, and usually one wants to switch either all on or all off.

**Comment by elbaraim** [comment]
codacy fails @yannikschaelte

**Comment by yannikschaelte** [comment]
> codacy fails @yannikschaelte

it only checks some branches

**Pull request merged**